dist/mention.js

```diff
@@ -145,7 +159,7 @@ angular.module('ui.mention').controller('uiMention', ["$element", "$scope", "$at
   this.decode = function () {
     var value = arguments.length <= 0 || arguments[0] === undefined ? ngModel.$modelValue : arguments[0];
 
-    return value ? value.replace(/@\[([\s\w]+):[0-9a-z-]+\]/gi, '$1') : '';
+    return value ? value.replace(this.decodePattern(), '$1') : '';
```

Comment:
I chose between function and property. I looked at the function `this.encode` and I think, that `this.decodePattern` is similar to `this.encode` logic.